manifests/2.5.0/opensearch-dashboards-2.5.0.yml

```diff
@@ -22,6 +22,12 @@ components:
     working_directory: dashboards-notifications
     ref: 2.x
   - name: observabilityDashboards
-    repository: https://github.com/opensearch-project/observability.git
-    working_directory: dashboards-observability
+    repository: https://github.com/opensearch-project/dashboards-observability.git
+    ref: 2.x
+  - name: reportsDashboards
+    repository: https://github.com/opensearch-project/dashboards-reporting.git
```

Comment:
This repo is still not public yet therefore cause the manifest CI checks failed.

Reply:
This repo is public now

Reply:
@zelinh Can you please re-run the failing tests?

Reply:
@mengweieric Could you rebase to resolve the conflicts?